src/plugins/gui/RewardsCardPlugin.tsx

```diff
@@ -30,7 +30,7 @@ export const makeRewardsCardPlugin: FiatPluginFactory = async params => {
   const { showUi, account, guiPlugin } = params
   const { pluginId } = guiPlugin
 
-  const SUPPORTED_ASSETS: EdgeTokenId[] = ['bitcoin', 'bitcoincash', 'dash', 'dogecoin', 'litecoin'].map(pluginId => ({ pluginId }))
+  const SUPPORTED_ASSETS: EdgeTokenId[] = ['bitcoin', 'bitcoincash', 'dogecoin', 'litecoin'].map(pluginId => ({ pluginId }))
```

Comment:
Optional: We could utilize the bool in the wallet constants instead of duplicating the data

```const SUPPORTED_ASSETS: EdgeTokenId[] = []
for (const pluginId of Object.keys(SPECIAL_CURRENCY_INFO)) {
    const { displayIoniaRewards = false } = SPECIAL_CURRENCY_INFO[pluginId] ?? {}
    if (displayIoniaRewards) SUPPORTED_ASSETS.push({ pluginId })
  }```